apis/placement/v1/clusterresourceplacement_types.go

```diff
@@ -500,7 +500,7 @@ type RollingUpdateConfig struct {
 	// Absolute number is calculated from percentage by rounding up.
 	// We consider a resource unavailable when we either remove it from a cluster or in-place
 	// upgrade the resources content on the same cluster.
-	// This can not be 0 if MaxSurge is 0.
+	// The minimum of MaxUnavailable is 1 to avoid rolling out stuck during in-place resource update.
 	// Defaults to 25%.
 	// +kubebuilder:default="25%"
 	// +kubebuilder:validation:XIntOrString
```

Comment:
Does this work?

+kubebuilder:validation:ExclusiveMinimum:true
+kubebuilder:validation:Minimum:0

Reply:
I dobut it as this is not an int